web/js/roomController.js

```diff
@@ -28,8 +28,6 @@ PhoneNumberController, ResizeSensor, maxUsersPerRoom */
 
   let { userName } = window;
   let roomURI = null;
-  let resolutionAlgorithm = null;
```

Comment:
Can we remove '/js/helpers/resolutionAlgorithms.js' from the list of `modules` loaded (see line 747)?

Reply:
Im not even sure if i totally follow whats the purpose of that file. If you have a good reason to do so, i can do it for sure